data/campaigns/tutorial/scenarios/02_Tutorial_part_2.cfg

```diff
@@ -871,10 +871,9 @@ A full list of abilities and weapons specials, along with traits, may be found i
         [filter]
             side=1
             race=elf
-            level=2
-            [or]
-                level=3
-            [/or]
```

Comment:
I'm not sure if it counts as a bug, but I do know it would be really easy to implement. In fact, range syntax would also be really easy implement. There are also a number of other keys for which comma-separated lists mean nothing. Perhaps it should be evaluated for them as well.